spec/integration/thread_spec.rb

```diff
@@ -2,6 +2,15 @@
 
 RSpec.describe HelpScout::Thread do
   let(:conversation_id) { ENV.fetch('TEST_CONVERSATION_ID') }
+  let(:thread_id) { ENV.fetch('TEST_THREAD_ID') }
+
+  describe '.get' do
```

Comment:
Feels like our getable shared example could be expanded to handle multiple args, but since I think this is the first instance of it, I'm cool with saving that for later.